src/redux/actions/logOut.actions.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-undef */
+/* eslint-disable import/prefer-default-export */
+export function logOutUser() {
```

Comment:
Hi @EmyRukundo, I haven't seen where you are using this function. I'm wondering why you have two logout actions and you are using only one of them.